spec/livefeed/result_spec.rb

```diff
@@ -2,17 +2,31 @@
 
 module Twingly::LiveFeed
   describe Result do
-    it { should respond_to :posts }
-    it { should respond_to :ts }
-    it { should respond_to :from }
-    it { should respond_to :number_of_posts }
-    it { should respond_to :max_number_of_posts }
-    it { should respond_to :first_post }
-    it { should respond_to :last_post }
-    it { should respond_to :next_timestamp }
+    instance_methods = [
+      :posts,
+      :ts,
+      :from,
+      :number_of_posts,
+      :max_number_of_posts,
+      :first_post,
+      :last_post,
+      :next_timestamp,
+    ]
+
+    instance_methods.each do |method|
+      it { should respond_to method }
+    end
 
     context "before query has populated responses" do
       its(:posts) { should be_empty }
     end
+
+    describe "#inspect" do
+      subject { described_class.new.inspect }
+
+      instance_methods.each do |method|
+        it { should include(method.to_s) }
```

Comment:
I think we need something before and after the method name here. `posts` matches `number_of_posts` for example. Something like the following (disclaimer: I have no idea whether this works):

```suggestion
        it { should match(/ #{method}[,=]/) }
```

Reply:
Yeah, good catch! I was a bit hesitant to repeat the exact syntax here, but that looks reasonable.

Reply:
1c7200e